precice_config_graph/edges.py

```diff
@@ -59,3 +57,26 @@ class Edge(Enum):
     # The connection between read-data and participant it is part of
     READ_DATA__PARTICIPANT_PARENT_OF = "read-data_participant-parent-of"
     READ_DATA__CHILD_OF = "read-data_child-of"
+
+    # connection between data and its export node
+    DATA_EXPORT = "data-export"
+    # connection between participant and export node
+    PARTICIPANT_EXPORT = "participant-export"
```

Comment:
```suggestion
    EXPORT__CHILD_OF = "export_child-of"
```
to be in line with, for example, `read-data`. If we are to switch to undirected edges, then this would probably not be needed.